gradle/wrapper/gradle-wrapper.properties

```diff
@@ -14,9 +14,9 @@
 # limitations under the License.
 #
 
-#Tue Apr 09 12:25:52 BST 2019
+#Tue Aug 27 09:42:42 BST 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.3.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
```

Comment:
You could possibly upgrade to Gradle 5.6, that's what we are doing at the API level. However, if it causes build problems that need investigating, 5.4.1 should be fine.